### PR TITLE
Add moveit_runtime metapkg

### PR DESCRIPTION
**Pull request by 130s**
Addresses #9.

### CODE REVIEW

File: moveit_runtime/package.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  <author email="gm130s@gmail.com">Isaac I. Y. Saito</author>
+  <maintainer email="gm130s@gmail.com">Isaac I. Y. Saito</maintainer>
+
+  <license>BSD</license>
+  <url type="website">http://moveit.ros.org</url>
+  <url type="website">http://wiki.ros.org/moveit_runtime</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <run_depend>moveit_commander</run_depend>
```
Comment (davetcoleman) [inline moveit_runtime/package.xml]: is this python interface necessary or too high level?

File: moveit_runtime/package.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  <run_depend>moveit_core</run_depend>
+  <run_depend>moveit_planners</run_depend>
+  <run_depend>moveit_plugins</run_depend>
+  <run_depend>moveit_ros</run_depend>
```
Comment (davetcoleman) [inline moveit_runtime/package.xml]: moveit_ros should be separated into its child packages, and moveit_ros_visualization and moveit_benchmarks should not be included

**Review by davetcoleman — DISMISSED** [review]

**Comment by 130s** [comment]
Thanks, comments addressed.

**Comment by 130s** [comment]
ping

**Comment by 130s** [comment]
@v4hn Can I ask your review since you reacted to #9?

**Comment by v4hn** [comment]
Sorry, for the delay, I suppose you want to see this merged before the release...

As far as I know/can see `moveit_ros_robot_interaction` is only used by `moveit_ros_visualization`.
Because visualization is left out already, I would propose to leave out this one too.

Other than that, this request looks good to me. Feel free to merge afterwards.

**Comment by 130s** [comment]
@v4hn thanks for the review. Change made and merging.

**Comment by v4hn** [comment]
@130s Now that we merged the `moveit_runtime` package into the main moveit repo, shouldn't we remove package from here again?
I just got a "multiple packages with the same name" error trying to build a workspace with both repositories checked out.

**Comment by 130s** [comment]
I agree.

Sorry, although this isn't admin action, I somehow committed directly w/o opening PR https://github.com/ros-planning/moveit_metapackages/commit/f93107d6647244145d2c68a7fecea7f3469712f4

**Comment by v4hn** [comment]
On Fri, Feb 03, 2017 at 10:17:52AM -0800, Isaac I.Y. Saito wrote:
> I agree.
> 
> Sorry, although this isn't admin action, I somehow committed directly w/o opening PR

No need to insist on formalism as long as everyone know what's happening. :)
Thanks for cleaning this up!

**Pull request merged**